src/runtime/StaticStrings.h

```diff
@@ -108,6 +109,9 @@ namespace Escargot {
     F(StringIterator)             \
     F(Symbol)                     \
     F(SyntaxError)                \
+    F(Temporal)                   \
+    F(Now)                        \
+    F(Calendar)                   \
```

Comment:
To save memory allocated for rarely used strings, we lazily initiailze these strings.
`FOR_EACH_LAZY_INTL_STATIC_STRING` is an example for INTL features.
These lazy strings are initialized only when it is actually used.
I think that `Temporal` feature is not frequently used yet. So, how about defining Temporal related strings as lazy static strings as well?